storage/pom.xml

```diff
@@ -47,6 +46,21 @@
             <artifactId>zipkin-sender-kafka</artifactId>
             <version>${zipkin-sender-kafka.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.linecorp.armeria</groupId>
+            <artifactId>armeria</artifactId>
+            <version>0.90.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.5</version>
+        </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
```

Comment:
Think this is pulled in by armeria by default in case you don't want to deal with bumping the version